subprojects/core-api/src/main/java/org/gradle/api/provider/ValueSource.java

```diff
@@ -31,10 +31,11 @@
  * among others.
  *
  * <p>
- * Representing external sources as {@link ValueSource}s allows Gradle to
- * transparently manage <a href="https://gradle.github.io/instant-execution/">the work graph cache</a>as values obtained from those sources
- * change. For example, a build might run a different set of tasks depending on whether
- * the {@code CI} environment variable is set or not.
+ * Representing external sources as {@link ValueSource}s allows Gradle to transparently manage
+ * <a href="https://docs.gradle.org/current/userguide/configuration_cache.html" target="_top">the configuration cache</a>
```

Comment:
This should be a URL to _this version_'s user manual, rather than to the most recently released version's user manual.

Reply:
I did the same as I found in other javadoc comments linking to the user guide.
I don't know how to do that in a javadoc comment, any trick?